app/http.php

```diff
@@ -83,21 +84,67 @@
 App::setResource('logger', fn () => $register->get('logger'));
 App::setResource('algorithm', fn () => $register->get('algorithm'));
 
-// Balancing must NOT be registry. This has to run on every request
-App::setResource('balancing', function (Table $state, Algorithm $algorithm) {
-    $balancing = new Balancing($algorithm);
+// Balancer must NOT be registry. This has to run on every request
+App::setResource('balancerGroup', function (Table $state, Algorithm $algorithm, Request $request) {
```

Comment:
We might be able to just keep it simpler with naming it just `balancer`. This is our (only) balancer after all.